cmd/tools/db-seeder/go.mod

```diff
@@ -0,0 +1,71 @@
+module sc-bos-db-seeder
```

Comment:
This is not a valid module name; it would have to be `github.com/vanti-dev/sc-bos/cmd/tools/db-seeder` (but it shouldn't be a module anyway)

Reply:
Hi this was script to run locally until @mattnathan wanted it as part of the tools library. Between setting up and seeding the laptop for the smartshow and getting that to work, this PR was created to review any issues with seeding but wasn't intended to be merged into the main branch, originally. This is why there are issues with the module naming.

As discussed this morning, this functionality will be part of a separate repository for running a demo. 

I plan to have a CI that will run when merging to main in sc-bos, to push to a hosted backend/frontend, but also build MacOS and Windows executables and publish them on slack so anyone is able to download from anywhere to any laptop.